src/sentry/api/endpoints/organization_member/__init__.py

```diff
@@ -59,7 +59,9 @@ def get_allowed_org_roles(
 
     if member is None:
         try:
-            member = OrganizationMember.objects.get(user=request.user, organization=organization)
+            member = OrganizationMember.objects.get(
+                user_id=request.user.id, organization=organization
```

Comment:
Should we be using `organization_id` here too?

Reply:
It's ok in this case because the organization member endpoint is a region silo endpoint, but it could definitely still be an organization id.

Reply:
Might be easier for future us if we bias towards always using `organization_id` as it is less error prone and works 'everywhere'. I think its fine for that to be a task for 'future us' as well.